HydroModelBuilder/ModelInterface/flopyInterface/flopyInterface.py

```diff
@@ -391,27 +391,25 @@ def add_bc_to_target(self, bc_array, target):
         return target
     # End add_bc_to_river()
 
-    def cell_bc_to_3D_array_for_plot(self, boundary, bc_array, nper, use_final=False):
+    def cell_bc_to_3D_array_for_plot(self, boundary, bc_array, nper, val_pos):
```

Comment:
Should this be moved to flopy_viz?

Reply:
@ConnectedSystems, quite possibly. That would be cleaner.